source/MaterialXTest/MaterialXGenMsl/CompileMSL.h

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright (c) 2023 Apple Inc.
+// Licensed under the Apache License v2.0
+//
+
+#ifndef COMPILEMSL_H
+#define COMPILEMSL_H
+
+#if __APPLE__
+void CompileMSLShader(const char* pShaderFilePath, const char* pEntryFuncName);
```

Comment:
For consistency with existing MaterialX code, I'd recommend using the letter case `Msl` in this function, as well as in the name of this file (e.g. `CompileMslShader`, `CompileMsl.h`).

Reply:
Thanks for the suggestion. Made this change.